src/requestforattestation/RequestForAttestation.ts

```diff
@@ -61,6 +63,95 @@ function getHashRoot(leaves: Uint8Array[]): Uint8Array {
   return hash(result)
 }
 
+function compressNonceAndHash(nonceHash: any[]): any {
+  if (nonceHash.length === 0) {
+    return {
+      hash: nonceHash[0],
+    }
+  }
+  return {
+    nonce: nonceHash[0],
+    hash: nonceHash[1],
+  }
+}
+
+export function compressClaimHashTree(
+  rfa: IRequestForAttestation
```

Comment:
Change request:
Unify namings; mostly in the SDK `reqForAtt` is used over `rfa` (and by the way `rfa` occurences should be replaced, but that's another story!)
This comment is valid for other new `rfa` occurences in this PR :) Once again nitpicking, but naming consistency makes it easier for other devs!

Reply:
Yeah, I can update it all.

Reply:
I see it's updated 👍